installation_docs/manual/linux.md

```diff
@@ -44,7 +44,7 @@ cd CircuitVerse
      ```
 - [Node.js 16.x](https://nodejs.org/it/download)
      ```bash
-     curl -sL https://deb.nodesource.com/setup_16.x | sudo bash
+     curl -sL https://deb.nodesource.com/node_16.x | sudo bash
```

Comment:
I guess for the install command in linux.md this would not work you need to use something like 
```
sudo apt-get update && sudo apt-get install -y ca-certificates curl gnupg
curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | sudo gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
NODE_MAJOR=16
echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_$NODE_MAJOR.x nodistro main" | sudo tee /etc/apt/sources.list.d/nodesource.list
sudo apt-get update && sudo apt-get install nodejs -y
```

as provided in https://deb.nodesource.com/

Reply:
The codespaces build also seems to be failing, So that needs to be fixed.

Reply:
I am working on this sir , fixing code space issue